ntcore/src/main/native/cpp/Value.cpp

```diff
@@ -31,6 +31,19 @@ struct StringArrayStorage {
   std::vector<std::string> strings;
   std::vector<NT_String> ntStrings;
 };
+
+template <typename T>
+inline std::shared_ptr<T[]> AllocateArray(size_t nelem) {
+#if __cpp_lib_shared_ptr_arrays >= 201707L
+#if __cpp_lib_smart_ptr_for_overwrite >= 202002L
+  return std::make_shared_for_overwrite<T[]>(nelem);
+#else
+  auto data = std::make_shared<T[]>(nelem);
```

Comment:
Shouldn't this return?

Reply:
Yes, it should.  I guess we don't have a compiler combo that hits this case.